docs/guards.md

```diff
@@ -58,16 +67,23 @@ class InvoiceStateMachine(StateMachine):
     paid = State("paid")
     failed = State("failed")
 
+    paused = False
+    offer_valid = True
+
     pay = (
-        unpaid.to(paid, cond="payment_success")
-        | failed.to(paid)
-        | unpaid.to(failed)
+        unpaid.to(paid, cond="payment_success") |
+        unpaid.to(failed, validators="validator", unless="paused") |
+        failed.to(paid, cond=["payment_success", "offer_valid"])
```

Comment:
**praise:** Nice that you demonstrated that properties and attributes can be used directly.  

**suggestion**: Maybe we could add an explicit note at `dynamic dispatch` that this is possible.